mappings/net/minecraft/client/render/entity/ItemEntityRenderer.mapping

```diff
@@ -1,4 +1,8 @@
 CLASS net/minecraft/class_916 net/minecraft/client/render/entity/ItemEntityRenderer
+	FIELD field_32925 MIN_ITEMS_FOR_5_RENDERED I
+	FIELD field_32926 MIN_ITEMS_FOR_4_RENDERED I
+	FIELD field_32927 MIN_ITEMS_FOR_3_RENDERED I
+	FIELD field_32928 MIN_ITEMS_FOR_2_RENDERED I
```

Comment:
These names don't convince me at all, any suggestions would be appreciated. They may even be the max for 4, 3, 2 and 1 rendered items respectively, since `#getRenderedAmount` checks for 'more than' to set the rendered amount.
```
int i = 1;
if (stack.getCount() > 48) { // 32925
   i = 5;
} else if (stack.getCount() > 32) { // 32926
   i = 4;
} else if (stack.getCount() > 16) { // 32927
   i = 3;
} else if (stack.getCount() > 1) { // 32928
   i = 2;
}
```

Reply:
They are more accurate now, but still don't convince me